src/enzo/ReadParameterFile.C

```diff
@@ -1187,6 +1187,12 @@ int ReadParameterFile(FILE *fptr, TopGridData &MetaData, float *Initialdt)
     ret += sscanf(line, "NumberOfPythonSubcycleCalls = %"ISYM, &NumberOfPythonSubcycleCalls);
 #endif
 
+#ifdef USE_LIBYT
+    ret += sscanf(line, "NumberOfLibytCalls = %"ISYM, &NumberOfLibytCalls);
+    ret += sscanf(line, "NumberOfLibytTopGridCalls = %"ISYM, &NumberOfLibytTopGridCalls);
+    ret += sscanf(line, "NumberOfLibytSubcycleCalls = %"ISYM, &NumberOfLibytSubcycleCalls);
+#endif
+
```

Comment:
These look like useful parameters, they should be described on the doc page.

Reply:
@matthewturk, do you know what their usages are? I only see them defined here and set in the user (code-specific) parameter, but Enzo's yt frontend doesn't seem to use it.

Reply:
Just to provide metadata.

Reply:
@matthewturk Should we remove it?